pkgs/misc/translatelocally-models/models.json

```diff
@@ -0,0 +1,382 @@
+{
+  "models": [
+    {
+      "modelName": "Czech-English base",
+      "shortName": "cs-en-base",
+      "type": "base",
+      "src": "Czech",
+      "trg": "English",
+      "srcTags": {
+        "cs": "Czech"
+      },
+      "trgTag": "en",
+      "repository": "Bergamot",
+      "version": 1,
+      "API": 1,
+      "checksum": "3714539160d5b4dce3ce0d829939315e3daffeaff53647249cc6336d745c09f2",
+      "url": "https://data.statmt.org/bergamot/models/csen/csen.student.base.tar.gz",
```

Comment:
I see a risk of these FODs breaking.
I see you're listed upstream as a contributor. Is it possible to somehow version these URLs, or are the artifacts intended to stay fixed?

Reply:
Apologies for the late reply.

All the models are open source. The deployment script is this one: https://github.com/browsermt/students/blob/master/deploy.students.sh
If you submit a PR and modify it, translateLocally doesn't care what the naming scheme of the student models is.

Someone (IE me) would just have to rerun the deploy script to make the changes permanent.